### DISCUSSION TITLE
estimateUserOp error when using P256 precompiled in L2

### DISCUSSION THREAD

**Discussion by ChiHaoLu** [alchemyplatform/aa-sdk#1115] [Issues]
When using the RIP-7212 in L2 like Arbitrum Sepolia or Optimism Sepolia, even these L2s support P256 precompiled, the Alchemy Bundler still returns error for missing precompiled.

**Comment by 0xfourzerofour** [comment]
please make a ticket in the rundler repo 

https://github.com/alchemyplatform/rundler
